addons/main/functions/fnc_doShareInformation.sqf

```diff
@@ -32,6 +32,14 @@ if (isNull _target) then {
     _target = _unit findNearestEnemy _unit;
 };
 
+// custom handlers
+private _handlersReturn = true;
+{
+    private _handlerResult = [_unit, _target, _range, _override] call _x;
+    if (_handlerResult isEqualTo false) exitWith {_handlersReturn = false};
```

Comment:
as far as I know, this function may be called in a Scheduled Environment what would if the _x function returns nil throw a script error here.

Reply:
Is it also necessary to write the result into a variable?

Reply:
> as far as I know, this function may be called in a Scheduled Environment what would if the _x function returns nil throw a script error here.

This is why isEqualTo is being utilized here. Nil returns or anything other than false would basically be treated as a true.

> Is it also necessary to write the result into a variable?

Strictly speaking, no, but it makes it a bit more readable.

Reply:
isEqualTo is no fix for that because nil is handled as undefined and will throw an `Error Undefined variable in expression`.

Reply:
Should be handled with recent commit